pkg/cnbutils/user.go

```diff
@@ -0,0 +1,32 @@
+package cnbutils
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+func CnbUserInfo() (int, int, error) {
+	uidStr, ok := os.LookupEnv("CNB_USER_ID")
+	if !ok {
+		return 0, 0, errors.New("environment variable CNB_USER_ID not found")
+	}
+
+	gidStr, ok := os.LookupEnv("CNB_GROUP_ID")
+	if !ok {
+		return 0, 0, errors.New("environment variable CNB_GROUP_ID not found")
+	}
```

Comment:
the group id stays 1000 on the images. why not default to 1000 or ignore the group id?

Reply:
The group id is 1000 in Paketo just by their choice, however it is not enforced by the spec.